packages/mongodb-memory-server-core/src/MongoMemoryServer.ts

```diff
@@ -46,7 +46,7 @@ export interface MongoInstanceDataT extends StartupInstanceData {
   childProcess?: ChildProcess;
 }
 
-export default class MongoMemoryServer {
+export class MongoMemoryServer {
```

Comment:
👍
Named exports work very well with TS in VSCode. When you start typing the class name it can be auto-imported for you (will be added `import ...` in the file header).